src/Android/Xamarin.Android/Samples/Network Analysis/TraceUtilityNetwork/TraceUtilityNetwork.cs

```diff
@@ -286,7 +284,14 @@ private async void Trace_Click(object sender, EventArgs e)
             catch (Exception ex)
             {
                 _status.Text = "Trace failed...";
-                CreateDialog(ex.Message);
+                if (ex is ArcGISWebException && ex.Message == null)
+                {
+                    CreateDialog($"HResult: {ex.HResult}", ex.GetType().Name);
```

Comment:
Instead of showing the HResult, I recommend checking for an empty/null message and showing a generic 'trace failed' message.